### ISSUE TITLE
Batch norm bug - BaselineMethod, Multi-Task Synbols

### ISSUE THREAD

**Issue by lebrice** [lebrice/Sequoia#166]
```console
sequoia_sweep --setting multi_task --max_runs 100 --dataset synbols --nb_tasks 12 --method baseline --encoder resnet18 --database_path orion/multi_task_synbols_baseline.pkl --project synbols_debug --monitor_training_performance True
```

[error_multitask.pdf](https://github.com/lebrice/Sequoia/files/6173563/error_multitask.pdf)

**Comment by lebrice** [comment]
Just for more context:
This appears to be caused by the input image in synbols being only 32x32, which, when using a resnet18, produces representations with only one value per channel, which raises an error when trying to use BatchNorm layer on them.
